src/cloudant/database.py

```diff
@@ -1098,3 +1101,123 @@ def get_query_result(self, selector, fields=None, raw_result=False,
             return QueryResult(query, **kwargs)
         else:
             return query.result
+
+    def get_search_result(self, ddoc_id, index_name, query, **kwargs):
+        """
+        Retrieves the raw JSON content from the remote database based on the
+        search index on the server, using the kwargs provided as query
+        parameters.
+
```

Comment:
You mention using kwargs as query parameters.  You might also want to mention using `query` as well.  Just a thought.